src/index.html

```diff
@@ -1,13 +1,63 @@
 <!doctype html>
 <html lang="en">
-  <head>
+  <head class="header">
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
+    <title>Stars</title>
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="rating__stars">
+
+      <div class="stars stars--0">
```

Comment:
Redundant empty line between parent and child elements

Reply:
Done